### ISSUE TITLE
mdns/avahi for Chromecast devices (or similar)

### ISSUE THREAD

**Issue by mousemat86** [anatosun/plexamp-docker#12]
Is there any way to have chromecast devices show up in plexamp – currently the player cannot see any speakers on my network despite setting up avahi/mdns. I'm not sure what else I'm missing to get it working.

**Comment by anatosun** [comment]
Have you tried with `network_mode: host`?

**Comment by mousemat86** [comment]
Yes I've tried this. Not sure if Plexamp should be able to see my Chromecast speakers or not
